shared_settings.ini

```diff
@@ -0,0 +1,48 @@
+; Settings shared by the connection and endpoint nodes. Overrides
+; should be specified in `autopush_settings.ini` and
+; `autoendpoint_settings.ini`.
+
+; Enable verbose logging for WebSocket connections and HTTP requests.
+; This should be disabled in production.
+debug = false
+
+; The public hostname. Defaults to the system hostname if unspecified.
+hostname =
```

Comment:
This just uses `socket.gethostname()`...so I think that'll return the internal hostname? I can clarify.